frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -228,45 +227,140 @@ export const convertFromMultiplexResponse = (
   });
 };
 
-export const areAOEAnswersComplete = (
+export const AoeValidationErrorMessages = {
```

Comment:
Still working my way through, quick nit for now: would it not make sense to have `AoeValidationErrorMessages` in `aoeUtils.ts` apposed to `testCardForm.utils.tsx`?

Reply:
I think it makes sense in `TestCardForm.utils` since the `TestCardForm` component is what is actually doing the AOE validation. The code in `aoeUtils` is used by the individual AOE form components (perhaps it makes sense for `aoeUtils` to be renamed `aoeFormUtils` but I don't feel strongly one way or another about renaming that)